share/src/main/java/org/apache/kafka/server/share/fetch/ShareAcquiredRecords.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.server.share.fetch;
+
+import org.apache.kafka.common.message.ShareFetchResponseData.AcquiredRecords;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * The ShareAcquiredRecords class is used to send the acquired records and associated metadata.
+ */
+public class ShareAcquiredRecords {
+
+    public static final ShareAcquiredRecords EMPTY_SHARE_ACQUIRED_RECORDS = new ShareAcquiredRecords();
```

Comment:
it is used internally, so could you please change it from public to private?

Reply:
Done.